From a5bcbaf6cb50b044362e07f692098707b9301c5e Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Thu, 21 Nov 2024 17:35:27 +0200
Subject: [PATCH] Use 0.6.12 version of google-ai-generativelanguage

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 7aac23e..fc90fb7 100644
--- a/setup.py
+++ b/setup.py
@@ -42,7 +42,7 @@ else:
     release_status = "Development Status :: 5 - Production/Stable"
 
 dependencies = [
-    "google-ai-generativelanguage==0.6.10",
+    "google-ai-generativelanguage>=0.6.10",
     "google-api-core",
     "google-api-python-client",
     "google-auth>=2.15.0",  # 2.15 adds API key auth support
-- 
2.34.1

